src/node_builtins.cc

```diff
@@ -458,30 +457,33 @@ MaybeLocal<Value> BuiltinLoader::CompileAndCall(Local<Context> context,
              .ToLocal(&get_internal_binding)) {
       return MaybeLocal<Value>();
     }
-    arguments = {realm->process_object(),
-                 get_linked_binding,
-                 get_internal_binding,
-                 realm->primordials()};
+    Local<Value> arguments[] = {realm->process_object(),
+                                get_linked_binding,
+                                get_internal_binding,
+                                realm->primordials()};
+    return CompileAndCall(
+        context, id, arraysize(arguments), &arguments[0], realm);
   } else if (strncmp(id, "internal/main/", strlen("internal/main/")) == 0 ||
```

Comment:
Just a nit, feel free to ignore.

```suggestion
  }
  
  if (strncmp(id, "internal/main/", strlen("internal/main/")) == 0 ||
```